lib/db/connection.rb

```diff
@@ -18,7 +18,13 @@ def environment
     end
 
     def config
-      @config ||= DB::Config.new(environment).options
+      if environment == 'development'
+        db_file = File.join(__FILE__, '..', '..', '..', 'config', 'database_dev.yml')
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.